### PR TITLE
Add json event logs to log rotation integration test

### PR DESCRIPTION
**Pull request by chenwany**
### Description of changes
* Add json event logs to log rotation integration test

### Tests
* Integration test

### References
* Link to related PRs in other packages (i.e. cookbook, node).
* Link to documentation useful to understand the changes.

### Checklist
- [ ] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [ ] Check all commits' messages are clear, describing what and why vs how.
- [ ] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [ ] Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by davprat — DISMISSED** [review]
lgtm

File: tests/integration-tests/tests/log_rotation/test_log_rotation.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def test_log_rotation(
     common_logs = [
         {"log_name": "cloud-init", "log_path": "/var/log/cloud-init.log", "existence": True},
         {"log_name": "supervisord", "log_path": "/var/log/supervisord.log", "existence": True},
+        {"log_name": "supervisord", "log_path": "/var/log/parallelcluster/bootstrap_error_msg"},
```
Comment (delongmeng) [inline tests/integration-tests/tests/log_rotation/test_log_rotation.py]: why is log_name `supervisord` here for the bootstrap_error_msg log?

Comment (chenwany) [reply tests/integration-tests/tests/log_rotation/test_log_rotation.py]: Good catch! :) PR is updated

**Review by delongmeng — APPROVED** [review]
lgtm

**Review by davprat — APPROVED** [review]

**Pull request merged**